### PR TITLE
Style cleanup for `modules/uncertainty/R/*.R` scripts 

### PR DESCRIPTION
**Pull request by bpbond**
Ran through formatR::tidy_dir with arrow=TRUE and indent=2; made sure all if blocks have braces. Tried to make sure formatR didn’t make things too jaggy. Substituted `seq_len`, `seq_along`, and `paste0`, and `file.path` as necessary. Made sure all functions end with an explicit `return`.

### CODE REVIEW

File: modules/uncertainty/R/plots.R [hunk 1]
```diff
@@ -122,61 +118,52 @@ format.plot.input <- function(plot.inputs, convert.var, trait.order=c()){
...
+  saplot <- saplot + ## plot spline function
+    geom_line(aes(x, y), data = data.frame(x = post.x, y = sa.spline(post.x)), size = linesize) + 
+    ## plot points used to evaluate spline
+    geom_point(aes(x, y), data = data.frame(x = sa.sample, y = sa.spline(sa.sample)), 
+               size = dotsize) + # indicate median with larger point
+    geom_point(aes(x, y), data = data.frame(x = sa.sample[median.i], y = sa.spline(sa.sample[median.i])), 
+               size = dotsize * 1.3) + 
+    scale_y_continuous(limits = range(pretty(y.range)), 
+                       breaks = pretty(y.range, n = 3)[1:3]) + theme_bw() + ggtitle(trait.lookup(trait)$figid)
+  theme(axis.text.x = element_text(size = fontsize$axis),
```
Comment (istfer) [inline modules/uncertainty/R/plots.R]: `theme` seems not to be connected to the plot, not due to @bpbond's changes though

File: modules/uncertainty/R/run.ensemble.analysis.R [hunk 2]
```diff
@@ -14,85 +15,104 @@
...
+  if (is.null(variable)) {
+    if ("variable" %in% names(settings$ensemble)) {
+      var <- which(names(settings$ensemble) == "variable")
+      for (i in seq_along(var)) {
+        variable[i] <- settings$ensemble[[var[i]]]
       }
     }
   }
-  if(is.null(variable)) logger.severe("No variables for ensemble analysis!")
+  if (is.null(variable))
```
Comment (istfer) [inline modules/uncertainty/R/run.ensemble.analysis.R]: no `{` around if-block

File: modules/uncertainty/R/run.ensemble.analysis.R [hunk 3]
```diff
@@ -141,88 +168,108 @@ runModule.run.ensemble.analysis <- function(settings, ...) {
...
-  if(is.null(start.year)) start.year <- settings$ensemble$start.year
-  if(is.null(end.year)) end.year <- settings$ensemble$end.year
-  if(is.null(start.year) | is.null(end.year)) logger.severe("No years given for ensemble analysis!")
-
-  if(is.null(variable)) {
-    if("variable" %in% names(settings$ensemble)){
-      var <- which(names(settings$ensemble) == 'variable')
-      for(i in 1:length(var)){
-        variable[i] = settings$ensemble[[var[i]]]
+  if (is.null(start.year))
```
Comment (istfer) [inline modules/uncertainty/R/run.ensemble.analysis.R]: no `{` around this and the next two if-blocks

File: modules/uncertainty/R/run.sensitivity.analysis.R [hunk 4]
```diff
@@ -20,149 +21,185 @@
...
+    if (is.null(end.year)) {
+      end.year <- settings$sensitivity.analysis$end.year
+    }
+    if (is.null(start.year) | is.null(end.year)) {
+      logger.severe("No years given for sensitivity analysis!")
+    }
+    if (is.null(variable)) {
+      variable <- settings$sensitivity.analysis$variable
+    }
+    if (is.null(variable))
```
Comment (istfer) [inline modules/uncertainty/R/run.sensitivity.analysis.R]: `{` needed

File: modules/uncertainty/R/run.sensitivity.analysis.R [hunk 5]
```diff
@@ -20,149 +21,185 @@
...
+
 
 ##' @export
 runModule.run.sensitivity.analysis <- function(settings, ...) {
-  if(is.MultiSettings(settings)) {
+  if (is.MultiSettings(settings)) {
     return(papply(settings, runModule.run.sensitivity.analysis, ...))
   } else if (is.Settings(settings)) {
-    run.sensitivity.analysis(settings, ...) 
+    return(run.sensitivity.analysis(settings, ...))
```
Comment (istfer) [inline modules/uncertainty/R/run.sensitivity.analysis.R]: @mdietze do we want a `return` here?

File: modules/uncertainty/R/variance.R [hunk 6]
```diff
@@ -27,31 +29,32 @@ variance.stats <- function(x){
...
-  if(class(trait.samples) == 'list'){
-    trait.samples <- matrix(unlist(trait.samples), 
-                            ncol = length(names(trait.samples)))
+  if (class(trait.samples) == "list") {
+    trait.samples <- matrix(unlist(trait.samples), ncol = length(names(trait.samples)))
     colnames(trait.samples) <- names(splinefuns)
-    if(!is.null(maxn) & maxn < nrow(trait.samples)){
-      j <- sample(1:nrow(trait.samples), maxn)
+    if (!is.null(maxn) & maxn < nrow(trait.samples)) {
+      j <- sample(seq_len(nrow(trait.samples), maxn))
```
Comment (istfer) [inline modules/uncertainty/R/variance.R]: this is not what the previous code was trying to do

File: modules/uncertainty/R/run.sensitivity.analysis.R (hunk 5 above)
Comment (bpbond) [inline modules/uncertainty/R/run.sensitivity.analysis.R]: I wasn't sure about this; meant to flag it in commit message, sorry.

File: modules/uncertainty/R/variance.R (hunk 6 above)
Comment (bpbond) [inline modules/uncertainty/R/variance.R]: This will produce the same thing, but I'll revert the code.

**Comment by bpbond** [comment]
Thanks for the catches @istfer -

File: modules/uncertainty/R/plots.R (hunk 1 above)
Comment (istfer) [inline modules/uncertainty/R/plots.R]: @dlebauer what do you think about this one?

Comment (dlebauer) [inline modules/uncertainty/R/plots.R]: I agree- looks like a missing `+`

Comment (dlebauer) [inline modules/uncertainty/R/plots.R]: I agree- looks like a missing `+`

File: modules/uncertainty/R/run.sensitivity.analysis.R (hunk 5 above)
Comment (istfer) [inline modules/uncertainty/R/run.sensitivity.analysis.R]: I can take care of this in my PR later if need be

**Pull request merged**